resources/views/users/DispProf.blade.php

```diff
@@ -3,7 +3,7 @@
 <div class='container'>
```

Comment:
In this file please change file name. Must use `camelCase` format for blade name not `StudlyCase` format